integration_tests/object_detection/config.py

```diff
@@ -35,7 +37,35 @@ def make_scene(scene_id, img_path, label_path):
 
     chip_sz = 300
     img_sz = chip_sz
-    data = ObjectDetectionImageDataConfig(img_sz=img_sz, augmentors=[])
+
+    scenes = [
+        make_scene('od_test', get_path('scene/image.tif'),
+                   get_path('scene/labels.json')),
+        make_scene('od_test-2', get_path('scene/image2.tif'),
+                   get_path('scene/labels2.json'))
+    ]
+    scene_dataset = DatasetConfig(
+        class_config=class_config,
+        train_scenes=scenes,
+        validation_scenes=scenes)
+
+    chip_options = ObjectDetectionChipOptions(neg_ratio=1.0, ioa_thresh=1.0)
+
+    if not nochip:
```

Comment:
You don't have to change this, but double negatives in if statements are a pet peeve.

Reply:
That's fair